stubs/msal/__init__.pyi

```diff
@@ -0,0 +1,13 @@
+"""Type stubs for msal"""
+
+from typing import Any, Optional
+
+class ConfidentialClientApplication:
+    """MSAL Confidential Client Application"""
+
+    def __init__(
+        self, client_id: str, *, client_credential: Optional[str] = None, authority: Optional[str] = None, **kwargs: Any
```

Comment:
nit: using Optional here but | in other files, we haven't adopted an official standard but mb we should try going forward

Reply:
I like |